lib/cocoapods-core/yaml_helper.rb

```diff
@@ -291,21 +289,21 @@ def sorting_string(value)
         /0x[0-9a-fA-F]+/, # base 16 int
         /[-+]?(\.[0-9]+|[0-9]+(\.[0-9]*)?)([eE][-+]?[0-9]+)?/, # float
         /[-+]?\.(inf|Inf|INF)/, # infinity
-        /\.(nan|NaN|NAN)/, # NaN
+        /\.(nan|NaN|NAN)/ # NaN
```

Comment:
Odd - is this because Rubocop doesn't realize these are array-like variadic args?

Reply:
lib/cocoapods-core/yaml_helper.rb:294:26: C: [Correctable] Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call, unless each item is on its own line.
        /\.(nan|NaN|NAN)/, # NaN